src/pages/integrations/iterable/event/index.md

```diff
@@ -13,26 +12,34 @@ The Iterable integration supports Android, iOS and Web data. All event data is f
 
 
 ## Data Processing Notes
+
 Iterable will not accept data more than 24 hours old.
 
 ## Prerequisites
 
-In order to forward events to Iterable, you must have an account with Iterable.
+In order to forward events to Iterable, you must have an account with Iterable. You can sign into your Iterable account at [https://app.iterable.com/login/](https://app.iterable.com/login)
+
+### Create a Mobile API Key 
+
+Create a dedicated **Mobile API key** for the Iterable Event integration:
+
+1. In the Iterable UI, navigate to to Integrations -> API Keys
+2. Click the "NEW API KEY" button
+3. Provide a descriptive name such as "mparticle-event-integration"
+4. Select the [Mobile](https://support.iterable.com/hc/en-us/articles/360035018152-Iterable-s-iOS-SDK-#_5-create-a-mobile-api-key-in-iterable) type
+5. Click "CREATE"
+6. Save the value of the key created. You will not be able to view it again. This value is needed to configure the Iterable Event Integration in mParticle.
+
+<aside class="warning"> You should always use an Iterable <strong>Mobile API key</strong> for the Iterable Event Integration. The Mobile API key's permissions are scoped to the endpoints required for sending data to Iterable and using Iterable's mobile features. If you're implementing the <a href="/integrations/iterable/event/#iterable-kit-integration">Iterable Kit</a>, mParticle will send this API key to the client-side SDK. Using a Standard API key will expose access to your project's data.</aside>
 
-1. Sign into your Iterable account at [https://app.iterable.com/login/](https://app.iterable.com/login)
-2. Create your Iterable API keys by selecting Integrations -> API Keys
-3. Click the Create New API Key button
-4. Select to create a key of type [Mobile](https://support.iterable.com/hc/en-us/articles/360035018152-Iterable-s-iOS-SDK-#_5-create-a-mobile-api-key-in-iterable)
-5. Click Create
-6. Save the value in the Key column of the table.  This value is needed to configure Iterable in mParticle.
+### Create Push Integrations
 
-### 3. Create Push Integrations
-If you will be sending push registration events, you will need to provide the Iterable Push Integration names as part of the mParticle configuration.  This can be done from the Iterable platform:
+If you will be sending push registration events, you will need to provide the Iterable Push Integration names as part of the mParticle configuration. This can be done from the Iterable platform:
```

Comment:
```suggestion
If you will be sending push registration events, you will need to provide the Iterable Push Integration names as part of the mParticle configuration. This can be done from Iterable:
```